lib/ssl/src/ssl.erl

```diff
@@ -20,7 +20,7 @@
 
 %%
 
-%%% Purpose : Main API module for the SSL application that implements TLS and DTLS 
```

Comment:
lots of unnecessary changes in this module.
please help to revert the space changes it so it wont conflict like hell in future merges.